Modules/_struct.c

```diff
@@ -31,15 +31,19 @@ get_struct_state(PyObject *module)
 static struct PyModuleDef _structmodule;
 
 #define _structmodulestate_global get_struct_state(PyState_FindModule(&_structmodule))
```

Comment:
`PyState_FindModule` won't work with multi-phase init: https://docs.python.org/3.8/c-api/module.html#c.PyState_FindModule

Reply:
Yes, I'm aware of that. The first commit does not enable multiphase init. I'm currently working on a second commit that takes care of the global cache. Once the cache is in the module state I can get rid of the global state and enable multiphase init.

Reply:
Also see https://bugs.python.org/issue42404

Reply:
Neat!

Reply:
I got rid of the global cache state and ``PyState_FindModule``.